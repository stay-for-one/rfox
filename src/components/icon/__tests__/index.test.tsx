import { render, configure } from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'
import React from 'react'
import Icon from '../index'

configure({ adapter: new Adapter() })

describe('Icon tests', () => { 
    it('renders correctly', () => {
        const tree = render(
            <Icon type="star" size={42} spin/>
        )
        expect(tree).toMatchSnapshot()
    })
})