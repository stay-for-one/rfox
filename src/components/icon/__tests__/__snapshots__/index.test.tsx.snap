// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon tests renders correctly 1`] = `
initialize {
  "0": Object {
    "attribs": Object {
      "class": "icon iconfont star spin",
      "style": "font-size:42px",
    },
    "children": Array [],
    "name": "i",
    "namespace": "http://www.w3.org/1999/xhtml",
    "next": null,
    "parent": null,
    "prev": null,
    "root": Object {
      "attribs": Object {},
      "children": Array [
        [Circular],
      ],
      "name": "root",
      "namespace": "http://www.w3.org/1999/xhtml",
      "next": null,
      "parent": null,
      "prev": null,
      "type": "root",
      "x-attribsNamespace": Object {},
      "x-attribsPrefix": Object {},
    },
    "type": "tag",
    "x-attribsNamespace": Object {
      "class": undefined,
      "style": undefined,
    },
    "x-attribsPrefix": Object {
      "class": undefined,
      "style": undefined,
    },
  },
  "_root": [Circular],
  "length": 1,
  "options": Object {
    "decodeEntities": true,
    "normalizeWhitespace": false,
    "withDomLvl1": true,
    "xml": false,
  },
}
`;
